cmd/run.go

```diff
@@ -107,6 +109,34 @@ a commandline interface for interacting with it.`,
 			Left:  func() string { return "    init" },
 		}
 
+		// Load plugins
+		cliConf, err := getConfig(cmd.Flags())
+		if err != nil {
+			return err
+		}
+
+		plugins := []plugin.JavaScriptPlugin{}
+		for _, pluginPath := range cliConf.Plugins {
+			jsPlugin, err := lib.LoadJavaScriptPlugin(pluginPath)
+			if err != nil {
+				return err
+			}
+
+			// Run plugin setup and add it to the module tree
+			if err = jsPlugin.Setup(); err != nil {
+				return err
+			}
+
+			// TODO: does this belong here, or is it the module package's responsibility?
+			mods := jsPlugin.GetModules()
+			for path, api := range mods {
```

Comment:
This probably needs to be a method in the `modules` package, as the runner shouldn't care about this sort of thing.

Reply:
Changed it so that it's a method in the `modules` package. Feels much nicer there.